### ISSUE TITLE
Dalaran wow shop charged me 10 additional donate points

### ISSUE THREAD

**Issue by Valdanif** [dalaranwow/dalaran-wow#4377]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

I entered to dalaran wow website and bought the heirloom Charmed Ancient Bone Bow, which price was 5 donate points in website's shop, when I bought it,, dalaran wow charged 10 additional donate points for some reason. At first I had 22 donate points, but after I bought the heirloom I had 7 donate points. When I entered to the game server, in my mailbox I saw 3 identical Charmed Ancient Bone Bows, I clicked return mail for 2 of them because I didn't buy those two and I wanted my money back, but when I logged out and went to the website, dalaran wow didn't give me back my 10 donate gold. Please, I need those 10 donate gold back, I spent real money for them. Can you please, dalaran wow, return me my 10 donate points?, I would really appreciate it.

**Comment by MalkaviaDW** [comment]
Hello there,

Write to the support email: dalaran.community@gmail.com, please. And explain the situation there (including character's name). It will be solved as soon as possible.

Cheers!
